pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -72,6 +72,10 @@ func (configgen *ConfigGeneratorImpl) BuildClusters(env model.Environment, proxy
 		clusters = append(clusters, configgen.buildInboundClusters(env, proxy, instances, managementPorts)...)
 	}
 
+	// Add a blackhole cluster for catching traffic to unresolved routes
+	// DO NOT CALL PLUGINS for this cluster.
+	clusters = append(clusters, buildBlackholeCluster())
```

Comment:
Nit: Naming, can we call this a `DevNullCluster` or something instead of Blackhole.

Reply:
I'd prefer not. This is exactly a black hole; we could call it "Sink" too if we really didn't like black hole. But IMO dev null is unintuitive in this context.

@rshriram BTW "black hole" is two words, so we should call this BlackHoleCluster (I had to check when writing this up ;) )

Reply:
I would have also taken `NullCluster` or `EmptyCluster` @ZackButcher - just something slightly more descriptive than Black Hole 😀